cds_ils/migrator/api.py

```diff
@@ -114,8 +118,33 @@ def import_parents_from_file(dump_file, rectype, include):
                     record = import_record(parent, model, provider)
                     records.append(record)
                 elif rectype == "multipart" and has_volumes:
-                    record = import_record(parent, model, provider)
-                    records.append(record)
+                    existing_parent = None
```

Comment:
Can we refactor this code to avoid this deep nesting?
I am counting `6` nested levels and many if...